JASP-Engine/JASP/R/descriptives.R

```diff
@@ -195,6 +195,29 @@ Descriptives <- function(jaspResults, dataset, options) {
     }
     .descriptivesScatterPlots(jaspResults[["scatterPlots"]], dataset.factors, variables, splitName, options)
   }
+  
+  # Dot plots
+  if(options[['descriptivesDotPlot']]){
+    
+    if(is.null(jaspResults[["DotPlots"]])) {
+      jaspResults[["DotPlots"]] <- createJaspContainer(gettext("Dot Plots"))
+      jaspResults[["DotPlots"]]$dependOn(c("plotVariables", "splitby", "descriptivesDotPlot"))
```

Comment:
```suggestion
      jaspResults[["DotPlots"]]$dependOn(c("splitby", "descriptivesDotPlot"))
```
`plotVariables` is not in the options right? If so, depending on it has no effect so I'd just omit it.